src/compiler/utilities.ts

```diff
@@ -7067,4 +7074,8 @@ namespace ts {
             return false;
         }
     }
+
+    export function containsIgnoredPath(path: string) {
+        return some(ignoredPaths, p => stringContains(path, p));

```

Comment:
Does this need to look at canonical path? eg. look at `isIgnoredPath` and `isInPath` in sys.ts

Reply:
I did look at that, but I noticed that where this is used, we were already doing a `containsNodeModulesPathPart` check without calling `getCanonicalFileName` or anything.